docs/modules/release-notes/pages/all-releases/4_13_3S.adoc

```diff
@@ -0,0 +1,107 @@
+= Kobiton 4.13.3S release notes
+:navtitle: Kobiton 4.13.3S release notes
+
+_July 19, 2024_
+
+== New OS support
+
+kobiton 4.13.3S add support for the following:
+
+* iOS/iPadOS 18 Beta
+* Xcode 16 Beta
+
+== CoreDevice: Lightning mode support for iOS 17.4 devices and later without networking changes
+
+When Apple introduced some major changes with the rollout of iOS 17, it requires significant network configuration to support Lightning mode with Kobiton. The additional network requirements are hard to implement and sometimes pose a security risks for some organizations.

```

Comment:
Please replace this paragraph with the following:
When Apple introduced iOS 17, they rolled out new connectivity protocols that impacted how iOS 17 devices communicated with solutions like Kobiton. These protocols required special network configurations for Kobiton Lightning Mode deployments that were not always possible for customers to implement.

Reply:
Replaced